shiny/_main.py

```diff
@@ -172,11 +185,13 @@ def run_app(
     reload: bool = False,
```

Comment:
Because we are adding arguments to the middle of the positional args, I am inclined to add a `*` arg after `port`. Or even after `app`?

The followup args seem like they should always be named.

Reply:
Right, and realistically this is always called from the command line so they'll be named in that case anyway.